lms/views/lti_launches.py

```diff
@@ -99,8 +100,16 @@ def handle_lti_launch(request, token=None, lti_params=None, user=None, jwt=None)
     4. If a student or teacher launches a module item that has been configured
        as a canvas file
     """
-    lti_key = lti_params['oauth_consumer_key']
-    context_id = lti_params['context_id']
+    try:
+        lti_key = lti_params['oauth_consumer_key']
+    except KeyError:
+        raise MissingLtiLaunchParamError('Oauth consumer key is required for lti launch.')
+
+    try:
+        context_id = lti_params['context_id']
+    except KeyError:
+        raise MissingLtiLaunchParamError('Context Id is required for lti launch.')
```

Comment:
I thought this PR was just for `'oauth_consumer_key'`? I guess the PR title and desc should be amended, to say that it fixes two of the bugs at once?